src/util/isEventTarget.ts

```diff
@@ -0,0 +1,3 @@
+export function isEventTarget(sourceObj: any): sourceObj is EventTarget {
```

Comment:
are these functions(`isEventTarget`, `isHTMLCollection`, `isNodeList`) need to be exported with separate module? do we have usage other than `fromEvent`?